zfs_uploader/job.py

```diff
@@ -60,6 +61,11 @@ def filesystem(self):
         """ ZFS filesystem. """
         return self._filesystem
 
+    @property
+    def s3_prefix(self):
+        """ S3 key prefix. """
```

Comment:
Could you make the variable description uniform across the README and object definitions?